plugins/scanners/scanoss/src/test/kotlin/ScanOssScannerDirectoryTest.kt

```diff
@@ -174,4 +174,75 @@ class ScanOssScannerDirectoryTest : StringSpec({
         // This assertion checks that our Go file was sent to the API.
         includedFiles.any { it.contains("random-data-10-41-29.go") } shouldBe true
     }
+
+    "Scanner should obfuscate file paths when path obfuscation is enabled" {
```

Comment:
@nnobelis, please check the logic of these tests. I'm only skimming over the formalities here.

Reply:
LGTM